query-service/build.gradle.kts

```diff
@@ -11,23 +11,28 @@ plugins {
 dependencies {
   implementation(project(":query-service-factory"))
   implementation("org.hypertrace.core.grpcutils:grpc-server-utils:0.11.2")
-  implementation("org.hypertrace.core.serviceframework:platform-grpc-service-framework:0.1.47")
+  implementation("org.hypertrace.core.serviceframework:platform-grpc-service-framework:0.1.49")
   implementation("org.slf4j:slf4j-api:1.7.32")
   implementation("com.typesafe:config:1.4.1")
 
   runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.17.1")
   runtimeOnly("io.grpc:grpc-netty")
-  integrationTestImplementation("com.google.protobuf:protobuf-java-util:3.20.1")
+  integrationTestImplementation("com.google.protobuf:protobuf-java-util:3.20.3")
```

Comment:
Move to https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java-util/3.22.0

Reply:
Done